cypress/e2e/EnrollmentPage/StagesAndEventsWidget/StagesAndEventsWidget.feature

```diff
@@ -53,6 +53,13 @@ Feature: User interacts with Stages and Events Widget
         When you click New First antenatal care visit event
         Then you should navigate to Add new page #/enrollmentEventNew?enrollmentId=ek4WWAgXX5i&orgUnitId=DwpbWkiqjMy&programId=WSGAb5XwJ3Y&stageId=WZbXY0S00lP&teiId=yFcOhsM1Yoa
 
+
     Scenario: User can not go to Add new page if stage is not repeatable and there is event in the stage
         Given you open the enrollment page by typing #enrollment?programId=IpHINAT79UW&orgUnitId=UgYg0YW7ZIh&teiId=fhFQhO0xILJ&enrollmentId=gPDueU02tn8
         Then you should see the disabled button New Birth event
+
+    # Waiting for pipline to update DB/DB update for 39 and 41
```

Comment:
How about `2.38` DB update? Shouldn't the changes made in https://github.com/dhis2/capture-app/pull/3683 be tested against version `2.38`, until [DHIS2-17545](https://dhis2.atlassian.net/browse/DHIS2-17545) is completed? Thanks!

Reply:
Good catch. It seems like we have a pipeline problem with 38 also though, so I'm in favour of not spending any time on 38.